nmr_commons/tracking/Trajectory.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+import csv
+import copy
+
+
+class Trajectory:
+    """Trajectory class where trajectory is a list of point and each point is a tuple of two points in space"""
+    ####
+    # Constructor
+    def __init__(self, items=None, norm_items=None):
```

Comment:
`items` is not too explanatory name. Maybe `points`?
And `norm_items` is never used, should be removed.